Parquet/src/main/java/io/deephaven/parquet/ParquetFileWriter.java

```diff
@@ -68,19 +73,19 @@ public RowGroupWriter addRowGroup(long size) {
     }
 
     public void close() throws IOException {
+        final OutputStream os = Channels.newOutputStream(writeChannel);
+        os.write(ParquetFileReader.MAGIC);
```

Comment:
We must have been doing this somewhere else... right? How did anything read us at all?

Reply:
Parquet is read backwards.  You seek to the end of the file and find the metadata: at the end, magic number, right before the end, the size of the metadata, which tells you how far back to seek to read said metadata.   The metadata itself has offsets in the file for the different section.   So you read all backwards, were the file itself is telling you things are.   So I think "almost" nothing would notice the /initial/ magic number.